src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -54,7 +55,9 @@ const AnnotationActivity = (props: Props) => {
     };
 
     const handleOnSelect = () => {
-        onSelect(id);
+        const { location } = target;
+        const { id: fileVersionId } = file_version;
+        onSelect(id, { fileVersionId, location });
```

Comment:
Should we just pass up the whole annotation?

Reply:
👍